src/ripple/protocol/SecretKey.h

```diff
@@ -72,25 +75,25 @@ class SecretKey
     const_iterator
     begin() const noexcept
     {
-        return buf_;
+        return buf_.data();
     }
 
     const_iterator
     cbegin() const noexcept
     {
-        return buf_;
+        return buf_.data();
     }
 
     const_iterator
     end() const noexcept
     {
-        return buf_ + sizeof(buf_);
+        return buf_.data() + buf_.size();
```

Comment:
Why not `buf_.cend();`?

Reply:
Will do.

Reply:
Addressed in e71cfc80b1583e5055c9f4ed5828b2e7c7deeeb4